packages/storage/src/userStorage.ts

```diff
@@ -149,6 +164,44 @@ export class UserStorage {
     return des;
   }
 
+  /**
+   * txlSubscribe is used to listen for Textile events.
+   *
+   * It listens to all buckets for the user and produces an event when something changes in the bucket.
+   *
+   * TODO: try to make the event more granular so we can pick up specific files/folders
+   *
+   * @example
+   * ```typescript
+   * const spaceStorage = new UserStorage(spaceUser);
+   *
```

Comment:
You forgot to show the `await spaceStorage.initListener()` in this doc example.

Also, what do you think we just make an alternate static factory construct for `UserStorage`. Something like `await UserStorage.fromUser(spaceUser)` and it would init the listener as well. So user doesn't have to forget about the txlSubscribe bit.